index.html

```diff
@@ -4,61 +4,71 @@
         <meta charset="utf-8">
         <title>Birthdays app</title>
         <meta name="description" content="Display the birthday of the contacts">
-        
+
         <meta name="theme-color" content="#ff7d14">
         <meta name="viewport" content="width=device-width">
+        <link rel="stylesheet" href="css/style/root.css">
+        <link rel="stylesheet" href="css/style/action_menu.css">
+        <link rel="stylesheet" href="css/style/option_menu.css">
         <link rel="stylesheet" href="css/style/headers.css">
         <link rel="stylesheet" href="css/style/lists.css">
         <link rel="stylesheet" href="css/app.css">
 
-        <script type="text/javascript" src="js/app.js" defer></script>
-
-        <link rel="prefetch" type="application/l10n" href="data/locales.ini" />
+        <meta name="defaultLanguage" content="en-US">
+        <meta name="availableLanguages" content="en-US">
+        <link rel="localization" href="data/{locale}.properties">
```

Comment:
I believe something is wrong with this change for loading localization. On the Firefox OS 2.2 simulator, I don't get the French translation anymore.

Now, I've zero knowlegde of app translation. I just started from the localized app template (from Web IDE if I remember well...).